src/ui/routes/+page.svelte

```diff
@@ -34,13 +34,14 @@
 				</Typography>
 			{/if}
 
-			<Button variant="raised" color="secondary" href="/initialize/configure" id="init-cluster">
+			<Button variant="raised" color="secondary" href="/package/deploy?path=init" id="init-cluster">
```

Comment:
wouldn't this be `/package/init/deploy` to be consistent?

Reply:
Agree with need to be consistent

Reply:
resolved with latest changes